components/NewFooter /index.jsx

```diff
@@ -0,0 +1,28 @@
+import StyleGuide from '../Styleguide'
+import clsx from 'clsx'
+import styles from './styles.module.scss'
+
+const NewFooter = () => {
+    return (
+        //outer and inner explanation?//
+        <div className={styles.footer}>
+            <div className={styles.container}></div>
```

Comment:
Remove, add border and spacing (change margin to padding) directly into `.footer`